lib-3rd-party/pquery/gan_parser_html.php

```diff
@@ -836,4 +836,4 @@ protected function parse_hierarchy($self_close = null) {
 	}
 }
 
-?>
\ No newline at end of file
+?>
```

Comment:
We need to fix this, I think there is a white space and that could cause issues.